src/constants/routes.ts

```diff
@@ -172,3 +172,6 @@ export const getCommitQueueRoute = (projectId: string) =>
 
 export const getCommitRoute = (projectIdentifier: string) =>
   `${paths.commits}/${projectIdentifier}`;
+
+export const getCommitsRoute = (projectId: string) =>
```

Comment:
this functions looks the same as `getCommitRoute` aside from variable naming

Reply:
oops